pkg/filters/int.go

```diff
@@ -67,11 +72,11 @@ func (filter *IntFilter) Parse(operatorAndValues string) error {
 		case "!=":
 			filter.NotEqual = append(filter.NotEqual, val)
 		case ">":
-			if (filter.Greater == GreaterNotSetInt) || (val > filter.Greater) {
+			if (filter.Greater == maxIntVal) || (val > filter.Greater) {
```

Comment:
with the terminology used in out BPF code, this line would look like:
`if (filter.Min == minNotSet) || (val > filter.Min) {`

Reply:
This logic is completely changed further down the line anyway, but I think `defaultGreatest` or something might be a better name in the userspace context.